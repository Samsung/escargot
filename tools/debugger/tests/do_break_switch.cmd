break do_break_switch.js:26
b do_break_switch.js:38
b test
c
c
c
c
