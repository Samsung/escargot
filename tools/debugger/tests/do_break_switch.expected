Connecting to: localhost:6501
Connection created!!!
Stopped at tools/debugger/tests/do_break_switch.js:42
(escargot-debugger) break do_break_switch.js:26
Breakpoint 1 at tools/debugger/tests/do_break_switch.js:26 (in test() at line:21, col:1)
(escargot-debugger) b do_break_switch.js:38
Breakpoint 2 at tools/debugger/tests/do_break_switch.js:38 (in f() at line:36, col:1)
(escargot-debugger) b test
Breakpoint 3 at tools/debugger/tests/do_break_switch.js:24 (in test() at line:21, col:1)
(escargot-debugger) c
Stopped at breakpoint:2 tools/debugger/tests/do_break_switch.js:38 (in f() at line:36, col:1)
(escargot-debugger) c
Stopped at breakpoint:3 tools/debugger/tests/do_break_switch.js:24 (in test() at line:21, col:1)
(escargot-debugger) c
Stopped at breakpoint:1 tools/debugger/tests/do_break_switch.js:26 (in test() at line:21, col:1)
(escargot-debugger) c
Connection closed.
